eng/Build.props

```diff
@@ -1,6 +1,18 @@
 <Project>
   <Import Project="Common.props" />
 
+  <!--
+    Copied from the Arcade SDK but w/ a nuance for npm projects. If specific projects need to build separately,
+    override their metadata below.
+  -->
+  <ItemDefinitionGroup>
+    <ProjectToBuild>
+      <BuildInParallel Condition=" '%(Extension)' != '.npmproj' ">false</BuildInParallel>
```

Comment:
@rainersigwald why am I hitting an error using `%(Extension)` here❔ The [docs](https://docs.microsoft.com/en-us/visualstudio/msbuild/item-definitions?view=vs-2022#value-sources) say "Well-known metadata on an Item from an ItemDefinitionGroup" is a value source though a [later section](https://docs.microsoft.com/en-us/visualstudio/msbuild/item-definitions?view=vs-2022#use-conditions-in-an-itemdefinitiongroup) mentions "Only local metadata references are supported in conditions".

Do I need to do something cumbersome like add `%(MyExtension)` metadata using `%(Extension)` as the value then put `%(MyExtension)` in the Condition❔

Reply:
Hmm, that cumbersome approach works but ick.